development_workflow.md

```diff
@@ -0,0 +1,69 @@
+# Development / Contribution Guidelines
+
+## General Development Workflow in the Lab
+
+
+
+### Forking and Branching
```

Comment:
I'd go the other way around and rather move that guide from the Seattle docs repo into here.